proto/addressmetadata.proto

```diff
@@ -23,6 +53,8 @@ message Payload {
     int64 ttl = 2;
     // User specified data.  Presumably some conventional data determined by wallet authors.
     repeated MetadataField rows = 3;
+    // Set of clauses under which the key may be revoked
+    repeated RevocationClause revocation_clauses = 4;
```

Comment:
This isn't meaningful.  The revocations need to be cryptographically verifiable.

Reply:
Each clause defines a cryptographically verifiable condition.

Reply:
Yes I understand, but this is not meaningful.